AODN/AODN-WAVE-DM/BOM-WAVE-DM/bom_wave_library/common.py

```diff
@@ -92,12 +92,12 @@ def set_glob_attr(nc_file_obj, data, metadata):
     setattr(nc_file_obj, 'time_coverage_end',
             data.datetime.dt.strftime('%Y-%m-%dT%H:%M:%SZ').values.max())
     setattr(nc_file_obj, 'date_created', datetime.now().strftime("%Y-%m-%dT%H:%M:%SZ"))
-    setattr(nc_file_obj, 'local_time_zone', metadata['timezone'])
-    setattr(nc_file_obj, 'method', METHOD_COMMENT)
-    setattr(nc_file_obj, 'original_filename', metadata['original_filename'])
+    # setattr(nc_file_obj, 'local_time_zone', metadata['timezone'])
+    setattr(nc_file_obj, 'abstract', METHOD_COMMENT)
+    # setattr(nc_file_obj, 'original_filename', metadata['original_filename'])
```

Comment:
its' actually a pity we can store this info in the netcdf file. We could have mentioned it in a "comment" attribute, but I've realised it's not in the standard. We probably could add it as a recommended attribute.

Reply:
do you mean the original_filename?